src/main/java/com/ft/methodecontentcollectionmapper/model/ContentCollectionType.java

```diff
@@ -0,0 +1,42 @@
+package com.ft.methodecontentcollectionmapper.model;
+
+import com.ft.methodecontentcollectionmapper.exception.UnsupportedTypeException;
+import com.google.common.collect.ImmutableMap;
+import javax.xml.xpath.XPathExpressionException;
+
+public enum ContentCollectionType {
+  STORY_PACKAGE_TYPE, CONTENT_PACKAGE_TYPE;
+
+  private static ImmutableMap<String, ContentCollectionType> webTypeToContentCollectionTypeMap = ImmutableMap.<String, ContentCollectionType>builder()
+      .put("editorsChoice", ContentCollectionType.STORY_PACKAGE_TYPE)
+      .put("content-package", ContentCollectionType.CONTENT_PACKAGE_TYPE)
+      .build();
+
+  private static ImmutableMap<ContentCollectionType, String> contentCollectionTypeToContentUriSuffixMap = ImmutableMap.<ContentCollectionType, String>builder()
+      .put(ContentCollectionType.STORY_PACKAGE_TYPE, "story-package")
+      .put(ContentCollectionType.CONTENT_PACKAGE_TYPE, "content-package")
+      .build();
+
+  public String getCorrespondingContentUriSuffix() {
+    return contentCollectionTypeToContentUriSuffixMap.get(this);
+  }
+
+  public static ContentCollectionType fromWebType(String webType, String uuid)
+      throws XPathExpressionException {
+    ContentCollectionType contentCollectionType = webTypeToContentCollectionTypeMap.get(webType);
```

Comment:
Checking for the value null for key in the map will return null for contentCollectionType which is checked for null on line 27, I would leave it like this because it will throw UnsupportedTypeException for this case also (when webType is null), do you think there should be a special Exception throwing for this case?